### ISSUE TITLE
Conversion of atomic unit length to angstrom.

### ISSUE THREAD

**Issue by hongyi-zhao** [doug-101/ConvertAll-py#12]
`1 a.u = 0.529177249 angstrom`
See [here](https://www.unitconverters.net/length/a-u-of-length-to-angstrom.htm), but it seems ConvertAll can't do this conversion. Any hints?

**Comment by doug-101** [comment]
The conversion on ￼your link (UnitConverters.net) is wrong.

ConvertAll gives a result of 1 au = 1.496 x 10^21 angstroms, which agrees with other conversions on Google.

You do realize that au is an astronomical unit, the distance from the Earth to the sun.  The angstrom is a very small unit for measuring atoms and molecules.  There is no way that one au is anywhere near half of an angstrom.

-Doug

**Comment by hongyi-zhao** [comment]
No, we're discussing different unit systems. I mean atomic unit by a.u. and it can be converted correctly by [pint](https://github.com/hgrecco/pint.git):

```
$ pint-convert bohr angstrom
1 bohr = 0.529177210904(81) Å
$ pint-convert a_u_length angstrom
1 bohr = 0.529177210904(81) Å

```
